libs/apps/env.go

```diff
@@ -0,0 +1,41 @@
+package apps
+
+import (
+	"strconv"
+)
+
+type EnvVar struct {
+	Name  string
+	Value string
+}
+
+func (e *EnvVar) String() string {
+	return e.Name + "=" + e.Value
+}
+
+func GetBaseEnvVars(config *Config) []EnvVar {
```

Comment:
Can we assert on a subset of envvars in the acceptance test? for example: calling `env` and then filter.

Reply:
We can't really because they set for the app command to be run and not globally in OS.